x-pack/libbeat/management/api/enroll.go

```diff
@@ -13,11 +13,12 @@ import (
 )
 
 // Enroll a beat in central management, this call returns a valid access token to retrieve configurations
-func (c *Client) Enroll(beatType, beatVersion, hostname string, beatUUID uuid.UUID, enrollmentToken string) (string, error) {
+func (c *Client) Enroll(beatType, beatName, beatVersion, hostname string, beatUUID uuid.UUID, enrollmentToken string) (string, error) {
```

Comment:
In the future we may want to have some kind of `EnrollConfig` for this method as I guess that the input parameters are going to grow.

Reply:
Yep, I thought about passing beatInfo object here, but hopefully this won't change in a while